XIVSlothCombo/Combos/PvE/SGE.cs

```diff
@@ -416,7 +418,18 @@ protected override uint Invoke(uint actionID, uint lastComboMove, float comboTim
                         if (IsEnabled(CustomComboPreset.SGE_ST_DPS_Phlegma) && InCombat())
                         {
                             uint phlegma = OriginalHook(Phlegma);
-                            if (InActionRange(phlegma) && ActionReady(phlegma)) return phlegma;
+
+                            if (InActionRange(Phlegma) && ActionReady(Phlegma))
+                            {
+                                if (!Config.SGE_ST_DPS_Phlegma_Adv || GetRemainingCharges(Phlegma) == GetMaxCharges(Phlegma)) 
+                                    return phlegma;
+                                
+                                if (GetRemainingCharges(Phlegma) == 1 && 
+                                    GetCooldownChargeRemainingTime(Phlegma) <= Config.SGE_ST_DPS_PhlegmaThreshold)
```

Comment:
Does not check `Config.SGE_ST_DPS_Phlegma_Adv` only a sub-option of it.

Reply:
its the else branch of the adv-config option (or full stacks). instead of two ifs it could be if-else true